checkout.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <title>Adopt A Dog</title>
+</head>
+<body>
+  <div class="grid-container">
+    <section id="banner">
+      <div id="letterhead">
+        <img src="images/adoptadog-logo.svg" id="toplogo" alt= "profile of a dog in white">
+        <h1>Seattle's premiere dog adoption agency</h1>
+      </div>
+      <div id="cart">
+        <img src="images/cart-icon.svg" id="carticon" alt="a white cart logo"><p id="cartprice">$1234.56</p>
+        <div id="checkout-button">
+          <p>Checkout</p>
```

Comment:
I would suggest putting this between an <a> tag with a link to your checkout.html.  Could apply this change to all of your pages.